Created contact form

shinyenyi opened the pull request:
Implemented html code and css styling to form contact and footer

Miguel-Shinyenyi committed:
add footer images

Khaled-AlKhateeb committed:
add HTML contact-form and footer

Miguel-Shinyenyi committed:
Implemented styling for contact form and footer

shinyenyi committed:
Merge branch 'main' into created_contact_form

Miguel-Shinyenyi committed:
Implemented styling for contact form and footer

Miguel-Shinyenyi committed:
fix linter erros

levy002 commented on index.html:
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
+    <form class="my-form" action="https://formspree.io/f/mrgdbqpg" method="POST">
+      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
```
- [ ] Kindly make sure to add all validations as requested in the project requirements. Please add the `required` attribute which helps us to not submit empty form input. Also, add `maxlength="30"` validation to validate the maximum input character to be inserted in the name `input`. :wink:

levy002 commented on index.html:
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
+      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
+      <input class="contact-form-email" type="email" name="email" placeholder="Email address">
```
- [ ] Kindly make sure to add all validations as requested in the project requirements. Please add the `required` attribute which helps us not submit empty form input.

levy002 commented on index.html:
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
+      <input class="contact-form-email" type="email" name="email" placeholder="Email address">
+      <textarea class="contact-form-message" rows="3" name="message" placeholder="Enter text here"></textarea>
```
- [ ] Kindly add the validation for the maximum character for this `textarea` to `maxlength="500"`.:wink:

levy002 commented on styles.css:
```diff
@@ -533,3 +533,117 @@ button[enabled] {
+  margin-bottom: 24px;
+}
+
+.contact-form-message {
+  margin-bottom: 24px;
+  border: none;
+  padding: 20px;
+  background-color: #fbf8f7;
+}
```
- [ ] Kindly adjust the width of the `textarea` in order to meet the provided design. :wink: 

   - ### Your design
  
   ![image](https://user-images.githubusercontent.com/87197412/186627615-15fd4e5c-0414-416e-ba71-42ef6d4c783c.png)


    - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186627802-098cf614-05d2-44a3-8a6a-1b298505960b.png)

levy002 commented on styles.css:
```diff
@@ -533,3 +533,117 @@ button[enabled] {
+  background-color: #e5e5e5;
+}
+
+.contact-form-content {
+  margin-top: 108px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 70px;
+}
```
- [ ] Kindly align your paragraph correctly in order to look exactly like the provided design. :wink: 

   - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/186628622-2c029b77-edf2-451c-a709-7e7ca9d18cbb.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186628462-f93f96a4-c87f-4392-98c8-9745e16d2a1c.png)

levy002 reviewed (changes requested):
## Hi @shinyenyi  && @Khaled-AlKhateeb  ,  :wave: 
### I'm Levy, your PR reviewer

Excellent, you did a great job :+1:  
However, they are some issues that you need to address before you move on to the next project :wink:
![](https://media.giphy.com/media/nYYtQdLqwwlTe3ccKh/giphy.gif)
## Highlights
- Good job for using the correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 
_check my comments under review_
 -  [ ] I think it would be better if we add all contributors' information to your README file. :wink: 
 - [ ] Kindly make sure to add a good  PR description. It makes your repo looks more professional. wink

  ![](https://user-images.githubusercontent.com/87197412/184990532-da778ab4-c789-4136-90a9-629eedd969da.png)
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] On your readme, you have the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  Kindly, make sure that The link featured in that section should points to a file at the root of your project (the license file).   Just copy it from the mentioned template:

   ![image](https://user-images.githubusercontent.com/57421823/168004669-c3026da8-8102-4a87-bb49-f7a5a9a6abbf.png)

That way, you will have a professional repo, and your project will be well documented 💪 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Miguel-Shinyenyi committed:
Make required changes
-Added my contributors information
-Add all validations as requested in project requirements
-Adjust text area width
-Aling get in touch paragraph correctly

frank1738 reviewed (approved):
Hi @shinyenyi , 

Good work implementing the changes requested. Your project looks great!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![YesYouAreGIF (2)](https://user-images.githubusercontent.com/50941074/172881298-64e98170-87c5-4075-8a77-71d65d991f55.gif)
## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
